typescript/nextjs/src/substreams/util.ts

```diff
@@ -0,0 +1,9 @@
+export const objectToJsonString = (obj: any): string => {
+    /*
+        Workaround because JS built-in function is not able to convert BigInt
+        https://github.com/GoogleChromeLabs/jsbi/issues/30
+    */
+    BigInt.prototype.toJSON = function() { return this.toString() }
```

Comment:
Should be separated and called only once when the application starts, in the NextJS entrypoint.

There is no need to add `toJSON` to `BigInt` prototype on each call. Further more, in a Browser context, `BigInt` should be checked to ensure it's available. Shouldn't be a problem since I think it's compatible enough even nowadays, but I always feel we should show the best code practices possible.